packaging/hip-targets.cmake

```diff
@@ -86,7 +86,7 @@ set_target_properties(hip::host PROPERTIES
 # Create imported target hip::device
 add_library(hip::device INTERFACE IMPORTED)
 
-if(HIP_COMPILER STREQUAL "hcc")
+if(CMAKE_CXX_COMPILER MATCHES "(.*)hcc")
```

Comment:
This should be

if (HIP_RUNTIME STREQUAL 'HCC')

Reply:
Agreed. For hip::device target, it should be fine to only test for HIP_RUNTIME. Device compilation cannot be done using standard c/c++ compilers.